### PR TITLE
Get article by id comments

### PR DESCRIPTION
**Pull request by JohnDMurphy**
-Get comments by article id
-error test for bad path
-error test for no comments / no article id

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -201,4 +201,55 @@ describe('nc-be-news-app', () => {
       expect(body.msg).toBe('Route not found!');
     });
   });
+
+  describe.only('GET article/:article_id/comments', () => {
```
Comment (dentednerd) [inline __tests__/app.test.js]: Lose the .only!

File: __tests__/app.test.js [hunk 2]
```diff
@@ -201,4 +201,55 @@ describe('nc-be-news-app', () => {
...
+      const { body } = await request(app)
+        .get('/api/articles/100/comments')
+        .expect(404);
+
+      expect(body.msg).toBe(
+        'the given ID does not exist or does not have any comments'
+      );
+    });
+
+    it('Should give a 404 error if given id does not have any comments', async () => {
```
Comment (dentednerd) [inline __tests__/app.test.js:245]: Are you sure this should be a 404 and not a 204?

**Review by dentednerd — COMMENTED** [review]
Just one small removal and one thing to think about! The rest is looking great

File: __tests__/app.test.js (hunk 1 above)
Comment (JohnDMurphy) [reply __tests__/app.test.js]: done

**Pull request merged**